ecl/eclagent/eclagent.ipp

```diff
@@ -954,9 +954,17 @@ private:
         {
             container = _container;
         }
-
+        virtual unsigned getWorkflowId() const override
```

Comment:
minor/style: it would be helpful if the implementations of the ICodeContext methods were grouped together.

Personally, I like to group all methods of an interface implementations together under a comment for clarity of origin etc., e.g.:

```
        void setContainer(EclSubGraph * _container)
        {
            container = _container;
        }
        void setWfid(unsigned _wfid)
        {
            wfid = _wfid;
        }
// ICodeContext
        virtual IEclGraphResults * resolveLocalQuery(__int64 activityId) override
        {
            if ((unsigned __int64)activityId == container->queryId())
                return container;
            return ctx->resolveLocalQuery(activityId);
        }
        virtual unsigned getWorkflowId() const override
        {
            return wfid;
        }
```

Reply:
I think it's generally worthwhile adding other missing 'override' when adding/altering interface implementations too (i.e. on resolveLocalQuery in this case)